creator-node/test/audiusUsers.test.js

```diff
@@ -49,15 +49,15 @@ describe('test AudiusUsers with mocked IPFS', function () {
   it('successfully creates Audius user (POST /audius_users/metadata)', async function () {
     const metadata = { test: 'field1' }
     ipfsMock.add.twice().withArgs(Buffer.from(JSON.stringify(metadata)))
-    ipfsMock.pin.add.once().withArgs('testCIDLink')
+    ipfsMock.pin.add.once().withArgs('QmYfSQCgCwhxwYcdEwCkFJHicDe6rzCAb7AtLz3GrHmuU6')
```

Comment:
lmao we only call this in health check now 🤦 k whatever, will rip out of tests and health check separately